distributed/protocol/tests/test_protocol_utils.py

```diff
@@ -8,3 +10,13 @@ def test_pack_frames():
     frames2 = unpack_frames(b)
 
     assert frames == frames2
+
+
+def test_frame_split_is_configurable():
+    frame = b"1234abcd" * (2 ** 20)  # 8 MiB
```

Comment:
Can we add a small `assert` here to ensure/confirm that `frame` is 8 MiB? 

```python
In [1]: frame = b"1234abcd" * (2 ** 20) # 8 MiB

In [2]: from dask.utils import parse_bytes

In [3]: from dask.sizeof import sizeof

In [4]: assert sizeof(frame) == parse_bytes("8 MiB")
```

Reply:
Sure, happy to do that. I sort of figured that the test itself functioned as an assert (3*3 ~ 8 and 2*5 ~ 8), but an assertion is more explicit